modules/backup-and-restore/pages/restore-backup-same.adoc

```diff
@@ -0,0 +1,25 @@
+= Restore a Database Backup from the Same Cluster
+:sectnums:
+:description:
+
+This page walks you through the process of restoring a database from a backup that was created from the same cluster.
+
+Restoring a backup is an offline operation.
+Your database will be down for a short period of time during database restore.
+
+== Prerequisites
+* You have access to the TigerGraph Linux user account on your cluster.
+All commands must be run from the TigerGraph Linux user.
+* You have access to a backup that was created from the same cluster.
+
+== Procedure
+To restore a backup, run the following command:
+
+[.wrap,console]
+----
+$ gadmin backup restore <backup_tag>
```

Comment:
FYI: gadmin restore support the `-y` option to skip interactive prompt questions by selecting the default answer.

Reply:
I tried running this command - there was no interactive prompt?

Reply:
do you use the latest package?
it should look like this
```
❯ gadmin backup restore backup-2022-09-09T151724
[   Note] gadmin backup restore needs to reset TigerGraph system.
Are you sure? (y/N)
```